.github/dependabot.yml

```
@@ -3,5 +3,10 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     labels: ["A2-insubstantial", "B0-silent", "C1-low"]
+    # Handle updates for crates from github.com/paritytech/substrate manually.
+    ignore:
+      - dependency-name: "substrate-*"
+      - dependency-name: "sc-*"
+      - dependency-name: "pallet-*"
```

I'd add a few other prefixes:
```suggestion
      - dependency-name: "sp-*"
      - dependency-name: "frame-*"
      - dependency-name: "fork-tree"
      - dependency-name: "pallet-*"
```

---

Thanks @ordian!